schema.sql

```diff
@@ -3,9 +3,10 @@ CREATE TABLE `translate_event_actions` (
   `event_id` int(10) NOT NULL COMMENT 'Post_ID of the translation_event post in the wp_posts table',
   `original_id` int(10) NOT NULL COMMENT 'ID of the translation',
   `user_id` int(10) NOT NULL COMMENT 'ID of the user who made the action',
-  `action` ENUM('approve','create','reject','request_changes') NOT NULL COMMENT 'The action that the user made (create, reject, etc)',
-  `locale` varchar(10) NOT NULL COMMENT 'Locale of the translation',
+  `action` enum('approve','create','reject','request_changes') NOT NULL COMMENT 'The action that the user made (create, reject, etc)',
+  `locale` varchar(10) CHARACTER SET latin1 COLLATE latin1_swedish_ci NOT NULL COMMENT 'Locale of the translation',
```

Comment:
Was this a deliberate change of accidental? If deliberate, why `swedish_ci`? Just curious :)

Reply:
It's the exact same charset as the locale on the GlotPress tables. It's a MySQL standard (they are Swedish).